src/components/molecules/Visualizer/Engine/Cesium/Tileset/index.tsx

```diff
@@ -13,25 +26,109 @@ export type Property = {
     tileset?: string;
     styleUrl?: string;
     shadows?: "disabled" | "enabled" | "cast_only" | "receive_only";
+    edgeWidth?: number;
+    edgeColor?: string;
+    clippingPlaneCollection?: ClippingPlaneCollection;
```

Comment:
```suggestion
    EXPERIMENTAL_clipping?: EXPERIMENTAL_Clipping;
```

Reply:
This was fixed in [fix: clipping collection to experimental](https://github.com/reearth/reearth-web/pull/338/commits/bf8000046bbbfb2d545d64a68bd4a8c2b9e7e2a9)